src/views/v2/list/top-bar/datepicker.php

```diff
@@ -74,18 +45,20 @@ class="tribe-events-c-top-bar__datepicker-time"
 				</span>
 			</time>
 		<?php endif; ?>
-		<span class="tribe-events-c-top-bar__datepicker-separator">&mdash;</span>
-		<time
-			datetime="<?php echo esc_attr( date_i18n( 'Y-m-d', $selected_end_datetime ) ); ?>"
-			class="tribe-events-c-top-bar__datepicker-time"
-		>
+        <?php if ( $show_end ) : ?>
```

Comment:
you've got a mix of spaces and tabs in this file. can you check and fix them back to tabs? it looks like your tab = 4 spaces.

Reply:
could we also add a test for this case in the partials tests?